statsd_exporter/versions.py

```diff
@@ -1,6 +1,6 @@
 versions = [
     {
-        "product": "0.26.1",
+        "product": "0.27.0",
```

Comment:
According to the source of truth we would not update it. However given the less coupling I'm happy to merge anyway, but please update the GSheet in that case

Reply:
Done.